.github/workflows/container.yaml

```
@@ -0,0 +1,85 @@
+name: Container Image
+
+on:
+  # Event triggered upon completion of the `release.yaml` workflow
+  workflow_run:
+    workflows: ['Release']
+    types:
+      - completed
+
+  # Manual trigger
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'Version to build (e.g., 1.42.4)'
+        required: true
+        type: string
+
+  workflow_call:
```

Maybe won't hurt to add a bit of context to readers who might be less familiar with how `workflow_call` works at a glance?

```suggestion
  # `release.yaml` calls this workflow after it has published a new release and attached files.
  # NOTE: As `release.yaml` is triggered via a tag being pushed, this workflow inherits
  # that push event context (github.{event_name, event, ref_type, ref_name, ...})
  workflow_call:
```